internal/verifier/verifier.go

```diff
@@ -43,45 +70,53 @@ type ReproducibleProvenanceVerifier struct {
 
 // Verify verifies a given SLSA provenance file by running the build script in
 // it and verifying that the resulting binary has a hash equal to the one
-// specified in the subject of the given provenance file. If the hashes are
-// different returns an error, otherwise returns nil.
+// specified in the subject of the given provenance file.
```

Comment:
I suggest updating the comment accordingly, instead of removing it.

Reply:
Done.